wsf_scraping/spiders/gov_spider.py

```diff
@@ -10,20 +11,47 @@ class GovSpider(BaseSpider):
         'JOBDIR': 'crawls/gov_uk'
     }
 
+    def __init__(self, *args, **kwargs):
+        year_before = kwargs.get('year_before', False)
+        year_after = kwargs.get('year_after', False)
+        id = kwargs.get('uuid', '')
+
+        self.uuid = id
+        if year_before:
```

Comment:
How do you feel about ternary operators?
`self.year_before = year_before if year_before else ''`

Reply:
Hum, I guess it can't hurt

Reply:
purely a style thing, could make a good case arguing against readability of ternary

Reply:
In this case it's cleaner I think 👍